src/gowrapper/ckks/ciphertext.go

```diff
@@ -105,3 +125,30 @@ func lattigo_poly(ctxHandle Handle8, i uint64) Handle8 {
 	}
 	return marshal.CrossLangObjMap.Add(unsafe.Pointer(ctx.Value[i]))
 }
+
+//export lattigo_polyQPCiphertextQP
+func lattigo_polyQPCiphertextQP(ctQPHandle Handle8, i uint64) Handle8 {
+	ctxQP := getStoredCiphertextQP(ctQPHandle)
+	return marshal.CrossLangObjMap.Add(unsafe.Pointer(&ctxQP.Value[i]))
+}
+
+//export lattigo_printMetaCtQP
+func lattigo_printMetaCtQP(ctQPHandle Handle8) {
```

Comment:
i think it would be cleaner to add methods for getting the internal values of the metadata. wdyt?